e2e/test/scenarios/embedding/embedding-linked-filters.cy.spec.js

```diff
@@ -106,6 +107,84 @@ describe("scenarios > embedding > dashboard > linked filters (metabase#13639, me
       });
     });
 
+    it("works when both filters are enabled and their values are set through UI with auto-apply filters disabled", () => {
+      cy.get("@dashboardId").then(dashboard_id => {
+        const payload = {
+          resource: { dashboard: dashboard_id },
+          params: {},
+        };
+
+        cy.request("PUT", `/api/dashboard/${dashboard_id}`, {
+          auto_apply_filters: false,
+        });
+
+        visitDashboard(dashboard_id);
+        visitEmbeddedPage(payload);
+      });
+
+      cy.findByRole("heading", { name: nativeDashboardDetails.name });
+      cy.get(".Card").contains(nativeQuestionDetails.name);
```

Comment:
There is `getDashboardCard` e2e helper to avoid using the css class

Reply:
fixed